server/Makefile

```diff
@@ -767,11 +767,11 @@ endif
 	$(GO) mod tidy
 
 ifeq ($(BUILD_ENTERPRISE_READY),true)
-	cp $(BUILD_ENTERPRISE_DIR)/imports/imports.go channels/imports/
+	mv enterprise/external_imports.go.orig enterprise/external_imports.go
 endif
 
 vet: ## Run mattermost go vet specific checks
-	$(GO) install github.com/mattermost/mattermost-govet/v2@new
+	$(GO) install github.com/mattermost/mattermost-govet/v2@5fd47765f8f278ddebe8034b233efa1bf9cc4103
```

Comment:
This is temporary until https://github.com/mattermost/mattermost-govet/pull/38 is reviewed.